src/styles/app.css

```
@@ -0,0 +1,161 @@
+@import 'tailwindcss';
+
+/* ============================================================
+   Fonts — self-hosted Geist (latin + latin-ext subsets).
+   Only weight 400 is shipped today; heavier weights synthesize
+   from this regular cut. Add new @font-face blocks (and copy
+   the matching .woff2 into ./fonts/) when a real heavy cut is
+   needed.
+   ============================================================ */
+
+@font-face {
+  font-family: 'Geist';
```

Stylelint (17.9.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)